DATA605/project_description.py

```diff
@@ -4,44 +4,117 @@
 import DATA605.project_description as dprodesc
 """
 
+import logging
+import os
+
 import helpers.hgoogle_file_api as hgofiapi
+import helpers.hio as hio
 import helpers.hopenai as hopenai
+import pandas as pd
+
+_LOG = logging.getLogger(__name__)
 
-# Constants
+# Constants.
 GOOGLE_SHEET_URL = "https://docs.google.com/spreadsheets/d/1Ez5uRvOgvDMkFc9c6mI21kscTKnpiCSh4UkUh_ifLIw/edit?gid=0#gid=0"
-PROMPT_DOC_URL = "https://docs.google.com/document/d/1c4oyCmvb-r5FHHL0enRQEEfsQ-shR3F7uvFVXMdeJSg/edit?tab=t.0#heading=h.a4upg2qbwqkb"
-MARKDOWN_FILE_PATH = "DATA605_Projects.md"
+PROMPT_DOC_URL = """
+You are a college professor of Data science. In the next prompt I will give you a topic XYZ for a class project and you will write a description using bullet points for a college class project about implementing an example big data system in Python.
+
+The project should be related to ingesting and processing real-time data about bitcoin. The focus should be on the technology XYZ, using basic Python packages for anything else.
+
+The assignment requires to describe the basic functionalities of the package using examples and then a concrete project related to implementing something related to time series analysis.
+The complexity of the project is 1, where 1 is easy (it should take around 7 days) to develop, 2 is medium difficulty (it should take around 10 days to complete), 3 is hard (it should take 14 days to complete).
+
+The output should follow the template below
+Title:
+Difficulty: (1=easy, 3=difficult)
+Description
+Describe technology
+Describe the project
+Useful resources
+Is it free?
+Python libraries / bindings
+
+An example of a project where XYZ is “AWS Glue” is the following
+
+Title: Ingest bitcoin prices using AWS Glue
```

Comment:
Let's add more bullets / formatting to this and the template so that it's automatically a nice / readable markdown